portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/GenerateAPI.testcase

```diff
@@ -0,0 +1,259 @@
+			var token = ACApi.getTokenValue();
+		}
+
+		task ("Create a new export schedule via API") {
+			ACApi.createNewExportSchedule(
+				endDate = "${endDate}",
+				endTime = "${endTime}",
+				startDate = "${startDate}",
+				startTime = "${startTime}",
+				token = "${token}",
+				type = "${type}");
+		}
+
+		task ("Check if the API response is correct for the error case") {
+			ACApi.viewApiResponseForError(message = "${message}");
+		}
+	}
+
+	@description = "Feature ID: LRAC-10833 | Automation ID: LRAC-11106 | Test Summary: The request command has the wrong date format"
+	@priority = "3"
+	test RequestCommandHasWrongDateFormat {
+		var startDate = "2022-02-01";
+		var endDate = "2022-05-16";
+		var type = "page";
+		var message = "Both dates in range must be ISO 8601 compliant yyyy-MM-dd'T'HH:mm[:ss.SSS'Z']";
+
+		task ("Get the token value to create the schedule") {
+			var token = ACApi.getTokenValue();
+		}
+
+		task ("Run curl command with wrong date format") {
+			var curl = '''
+				http://localhost:8080/api/reports/export/${type}\?fromDate=${startDate}\&toDate=${endDate}
+				-H "Authorization: Bearer ${token}"
+				''';
+
+			var apiResponse = JSONCurlUtil.get("${curl}", "$");
+		}
+
+		task ("Check if the API response is correct for the error case") {
+			ACApi.viewApiResponseForError(
+				apiResponse = "${apiResponse}",
+				message = "${message}");
+		}
+	}
+
+	@description = "Feature ID: LRAC-10833 | Automation ID: LRAC-11105 | Test Summary: The request command does not contain 'fromDate' or 'toDate'"
+	@priority = "3"
+	test RequestCommandNotContainFromDateOrToDate {
+		var startDate = "1964-07-09";
+		var startTime = "15:36:00.000";
+		var endDate = "2017-06-15";
+		var endTime = "10:20:00.000";
+		var type = "account";
+		var message = "&quot;fromDate&quot; and &quot;toDate&quot; query parameters are mandatory and must be ISO 8601 compliant yyyy-MM-dd'T'HH:mm[:ss.SSS'Z']";
+
+		task ("Get the token value to create the schedule") {
+			var token = ACApi.getTokenValue();
+		}
+
+		task ("Run curl command without 'fromData' and 'toData'") {
+			var curl = '''
+				http://localhost:8080/api/reports/export/${type}
+				-H "Authorization: Bearer ${token}"
+				''';
+
+			var apiResponse = JSONCurlUtil.get("${curl}", "$");
+		}
+
+		task ("Check if the API response is correct for the error case") {
+			ACApi.viewApiResponseForError(
+				apiResponse = "${apiResponse}",
+				message = "${message}");
+		}
+
+		task ("Run curl command without 'fromDate'") {
+			var curl = '''
+				http://localhost:8080/api/reports/export/${type}\?toDate=${endDate}T${endTime}Z
+				-H "Authorization: Bearer ${token}"
+				''';
+
+			var apiResponse = JSONCurlUtil.get("${curl}", "$");
+		}
+
+		task ("Check if the API response is correct for the error case") {
+			ACApi.viewApiResponseForError(
+				apiResponse = "${apiResponse}",
+				message = "${message}");
+		}
+
+		task ("Run curl command without 'toData'") {
```

Comment:
Just some minor typos I believe it should be toDate. Besides that great job on automating this story!

Reply:
Good catch, thanks Josh 😄